### PR TITLE
Morse Code Task

### PR DESCRIPTION
**Pull request by Chimoneg27**
### For this feature me and my partner did the following

- Used the proper gitflow
- Added the required linters
- Made sure to remove linter errors
- Created our gem files
- Solved the morse code challenge using ruby methods and pair programming

### CODE REVIEW

**Review by zeff96 — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW 🛑
Hi @Chimoneg27 and @Fombi-Favour , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

invalid

## Invalid Code Review Request


You've turned in a project, but it's missing a professional README file! 😬 Let's make sure to add a README documentation to your project, using the Microverse README template 📝📋, and let's customize it to perfectly match and enhance your project. 🛠🚀

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: morse_code_decoder.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  end
+  print @current_word
+end
+
+def decode_message(message)
+  @words = message.split
+  @words.each do |word|
+    decode_word(word)
+  end
+end
```
Comment (codepantha) [inline morse_code_decoder.rb]: - [ ] There seems to be a bug 🐛 with your `decode_message` method as it doesn't work as expected.
For instance, `p decode_message(".-   -... --- -..-   ..-. ..- .-.. .-..    --- ..-.    .-. ..- -... .. . ...")` or `decode_message(".-   -... --- -..-   ..-. ..- .-.. .-..    --- ..-.    .-. ..- -... .. . ...")` should return "A BOX FULL OF RUBIES" but instead it returns the message without spaces between words and a bunch of characters as well. Please try to fix it 👇 
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @Fombi-Favour 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ No linter errors 🟢 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**